packages/main/src/Icon.ts

```diff
@@ -118,6 +119,34 @@ const PRESENTATION_ROLE = "presentation";
  */
 @event("click")
 class Icon extends UI5Element {
+	/**
+	 * Defines the component semantic color.
+	 *
+	 * <br><br>
+	 * <b>The available values are:</b>
+	 *
+	 * <ul>
+	 * <li><code>Contrast</code></li>
+	 * <li><code>Critical</code></li>
+	 * <li><code>Default</code></li>
+	 * <li><code>Information</code></li>
+	 * <li><code>Marker</code></li>
+	 * <li><code>Negative</code></li>
+	 * <li><code>Neutral</code></li>
+	 * <li><code>NonInteractive</code></li>
+	 * <li><code>Positive</code></li>
+	 * <li><code>Tile</code></li>
+	 * </ul>
+	 *
+	 * @type {sap.ui.webc.main.types.IconColor}
+	 * @name sap.ui.webc.main.Icon.prototype.design
+	 * @defaultvalue "Default"
+	 * @public
+	 * @since 1.9.2
+	 */
+	@property({ type: IconColor, defaultValue: IconColor.Default })
+	color!: IconColor;
```

Comment:
Also, after brief discussion with the team, we think "design" and "IconDesign" are more suitable for the use-case

Reply:
I initially implemented it as design, but me and our acc expert decided it's better to keep it color since it's not acc related and it's only visual preference.

I am okay with both variants.

Reply:
Our reasoning for the name `color` was meant to align with the UI5 API names and to self explain that the value is suitable for any color related properties of the icon. For example in UI5 apart from the `color` property there are also `backgroundColor`, `activeColor`, `hoverColor`, etc. And as Jordan said to be simpler to think about the APIs. In case there is a conflict with the CSS proeprty `color` I'm also for to name it `design`.